NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -625,6 +657,15 @@ private void TargetRichTextBox_LinkClicked(object sender, LinkClickedEventArgs e
                 linkClickEvent(this, linkText, logEvent);
             }
         }
+        
+        /// <summary>
+        /// Used to detach the control manually from the target
+        /// </summary>
+        public void DetachControl()
```

Comment:
I'm OK with this one

Reply:
I see this is:

```
        /// <summary>
        /// Closes the target and releases any unmanaged resources.
        /// </summary>
        protected override void CloseTarget()
        {
            DetachFromControl();
        }

```

But maybe a bit unhandy (called when closing/shutting down the full nlog config)

Reply:
What would be the purpose to having it attached once the target is closed??

Reply:
I will be now detached after the close of the target.

Reply:
So it's already possible, that's my point